hack/values_ekstcl.yaml

```diff
@@ -0,0 +1,8 @@
+enableVolumeSnapshot: true
+controller:
+  logLevel: 5
+node:
+  logLevel: 5
+serviceAccount:
+  controller:
```

Comment:
with latest helm chart, should be controller.serviceAccount not serviceAccount.controller similar to the EBS one.
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/hack/values_eksctl.yaml

Also the enalbeVolumeSnapshot line can be removed